### PR TITLE
Pr auc metric

### PR DESCRIPTION
**Pull request by azikmsu**
Before submitting a pull request, please do the following steps:

1. Read [instructions for contributors](https://catboost.ai/docs/concepts/development-and-contributions.html).
2. Run `ya make` in catboost folder to make sure the code builds.
3. Add tests that test your change.
4. Run tests using `ya make -t -A` command.
5. If you haven't already, complete the [CLA](https://yandex.ru/legal/cla/).

### CODE REVIEW

File: catboost/private/libs/options/enums.h [hunk 1]
```diff
@@ -177,6 +177,7 @@ enum class ELossFunction {
     WKappa,
     LogLikelihoodOfPrediction,
     NormalizedGini,
+    PrecisionRecallAUC,
```
Comment (nikitxskv) [inline catboost/private/libs/options/enums.h:180]: Давай заменим на `PRAUC`

**Comment by nikitxskv** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
CLA already signed

**Comment by arcadia-devtools** [comment]
@nikitxskv, internal review request created: [1247069](https://a.yandex-team.ru/review/1247069)

**Comment by nikitxskv** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
@nikitxskv, internal review request updated: [1247069](https://a.yandex-team.ru/review/1247069)

**Comment by nikitxskv** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
@nikitxskv, internal review request updated: [1247069](https://a.yandex-team.ru/review/1247069)

**Comment by nikitxskv** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
@nikitxskv, internal review request updated: [1247069](https://a.yandex-team.ru/review/1247069)

**Pull request merged**